README.md

```diff
@@ -28,118 +28,96 @@
+  - Dev Ex: Support the IPFS technical community through documentation, contributor experience, API ergonomics and tooling
+- **Project:** Support team functioning, prioritization, and day-to-day operations
 
-The IPFS project is the collective work of serveral focused teams, called Working Groups. Each group defines its own roadmap with tasks and priorities derived from the main IPFS Project Priority for 2019 described above. You can drill-down into each roadmap to get the full picture, or jump to the [2019 epics](#2019-epics) section below for the highlights.
 
-- [Project WG](2019%20WG%20roadmaps/WG_PROJECT.md)
-- [JS Core](2019%20WG%20roadmaps/WG_JS_CORE.md)
-- [Go Core](2019%20WG%20roadmaps/WG_GO_CORE.md)
-- [GUI](2019%20WG%20roadmaps/WG_GUI.md)
-- [Cluster](2019%20WG%20roadmaps/WG_CLUSTER.md)
-- [IPFS Infrastructure](2019%20WG%20roadmaps/WG_INFRASTRUCTURE.md)
-- [Community](2019%20WG%20roadmaps/WG_COMMUNITY.md)
-- [Integration in Web Browsers](2019%20WG%20roadmaps/WG_INTEGRATION_IN_WEB_BROWSERS.md)
-- [Dynamic Data & Capabilities](2019%20WG%20roadmaps/WG_DYNAMIC_DATA_AND_CAPABILITIES.md)
-- [Decentralized Data Stewardship](2019%20WG%20roadmaps/WG_DECENTRALIZED_DATA_STEWARDSHIP.md)
+## 2020 Epics
 
-## 2019 IPLD & libp2p Roadmaps
+We've expanded our 2020 Priority into a list of **Epic Endeavours** that give an overview of the primary targets IPFS has for 2020 H1. If you are pumped about these Epics and want to help, you can get involved! See the call to action (CTA) for each section below.
 
-You can find the IPFS sister projects' Roadmaps at:
+### 1. Build IPFS dev capacity and velocity 
+In order to achieve our content routing goal for 2020 H1, we need to level up our own leverage, coordination, velocity, and planning as a project to ensure all contributors spend their time and energy effectively. This includes a few different dimensions:
 
-- [libp2p](https://docs.google.com/document/d/1Rd4yNw1TNQBvfRrKeEMSTseb6fvPzS-C--obOn0nul8/edit)
-- [IPLD](https://github.com/ipld/roadmap)
+- Integrate research via the [ResNetLab](https://research.protocol.ai/posts/201912-resnetlab-launch/) into our design practice to ensure our work builds on the knowledge and experience of leading researchers in our fields
+- Empower new contributors in the IPFS ecosystem through [DevGrants](https://github.com/ipfs/devgrants) and collaborations to upgrade and extend IPFS to solve new problems
+- Invest in developer tooling, automation, and fast feedback loops to accelerate experimentation and iteration
+- Upgrade project planning and management within and between working groups to ensure we define, estimate, track and unblock our work efficiently
+- Focus our attention on fewer things to improve completion rate and reduce churn, saying "no" to more _nice-to-have_ projects in order to allocate energy and attention to the most important work
```

Comment:
Hm, we're often just hitting pause or using tools like grants, instead of a hard no. Maybe finish with:

", allocating energy and attention to the most important work and either pausing or stopping funding lower priority projects"

Reply:
added a note about finding champions for lower priority projects